packages/sw-precaching/src/lib/revisioned-cache-manager.js

```diff
@@ -143,6 +156,14 @@ class RevisionedCacheManager {
 
       event.waitUntil(
         this._performInstallStep()
+        .then(() => {
+          // Closed indexedDB now that we are done with the install step
+          this._close();
+        }, (err) => {
```

Comment:
Would you mind switching to `.catch()`? I find the two-argument `.then()` to be much harder to follow.